pipelines/skylab/multiome/Multiome.changelog.md

```diff
@@ -1,3 +1,10 @@
+# 3.1.1 
+2024-01-30 (Date of Last Commit)
+
+* Added task GetNumSplits before FastqProcess ATAC task to determine the number of splits based on the bwa-mem2 machine specs
+* Added an error message to the BWAPairedEndAlignment ATAC task to ensure that the number of splits equal the number of ranks
```

Comment:
Where was this error message added? In the BWA task?

Reply:
It was added to intel's code for distributed bwa-mem2